simpleflow/command.py

```diff
@@ -527,3 +530,54 @@ def standalone(context,
     worker_proc.join()
     os.kill(decider_proc.pid, signal.SIGTERM)
     decider_proc.join()
+
+
+@click.option('--domain',
+              envvar='SWF_DOMAIN',
+              required=False,
+              help='Amazon SWF Domain.')
+@click.option('--workflow-id',
+              required=True,
+              help='ID of the workflow execution.')
+@click.option('--run-id',
+              required=False,
+              help='Run ID of the workflow execution.')
+@click.option('--scheduled-id',
+              required=False,
+              type=int,
+              help='Event ID when the activity has been scheduled.')
+@click.option('--activity-id',
+              required=False,
+              help='Activity ID of the activity you want to replay.')
+@cli.command('activity.rerun', help='Rerun an activity task locally.')
+def activity_rerun(domain,
+                   workflow_id,
+                   run_id,
+                   scheduled_id,
+                   activity_id):
+    # check params
+    if not activity_id and not scheduled_id:
+        logger.error("Please supply --scheduled-id or --activity-id.")
+        sys.exit(1)
+
+    # find workflow execution
+    try:
+        wfe = helpers.get_workflow_execution(domain, workflow_id, run_id)
+    except (swf.exceptions.DoesNotExistError, IndexError):
+        logger.error("Couldn't find execution, exiting.")
+        sys.exit(1)
+    logger.info("Found execution: workflowId={} runId={}".format(wfe.workflow_id, wfe.run_id))
+
+    # now rerun the specified activity
+    history = History(wfe.history())
+    history.parse()
+    func, args, kwargs, params = helpers.find_activity(
+        history, scheduled_id=scheduled_id, activity_id=activity_id
+    )
+    logger.debug("Found activity. Last execution:")
+    for line in json_dumps(params, pretty=True).split("\n"):
+        logger.debug(line)
+    logger.info("Will re-run: {}(*{}, **{})".format(func.__name__, args, kwargs))
```

Comment:
We can `logger.info(json.dumps(return_value))`, and accept a `--input` option to this command that would override the input extracted from the workflow execution history. In that case the main benefit of this command would be to avoid dealing with the function import, but fine for me :-)